src/transactions/TransactionUtils.cpp

```diff
@@ -15,39 +15,97 @@
 namespace stellar
 {
 
-LedgerTxnEntry
-loadAccount(AbstractLedgerTxn& ltx, AccountID const& accountID)
+LedgerKey
+getLedgerKey(AccountID const& accountID)
```

Comment:
I like those utils! I think those could even have shorter names, like just ledgerKey, it would make them into some kind of constructors (and I generally don't like 'get' prefixes)

Reply:
I would prefer to keep these as is and avoid confusion between `ledgerKey` and `LedgerKey`.